packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -333,116 +350,123 @@ class _GoogleMapState extends State<GoogleMap> {
     );
     _controller.complete(controller);
     _updateTileOverlays();
-    if (widget.onMapCreated != null) {
-      widget.onMapCreated(controller);
+    final onMapCreated = widget.onMapCreated;
```

Comment:
nits: specify types. (ditto below if decided to fix)
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic

Reply:
Done throughout.